automation/terraform/monitoring/README.md

```diff
@@ -0,0 +1,180 @@
+
+###### manual steps
+
+```
+    terraform apply -target module.o1testnet_alerts.docker_container.sync_alert_rules
+```
+
+**Note:** operation will sync provisioned alerts with exact match of alert file state (e.g. alerts removed from the alert file will be unprovisioned on Grafanacloud)
+
+**Warning:** Unfortunately terraform silences errors which cortextool is producing, for example when syncing rules. In order to debug sync rules process:
+- Obtain secrets (id,address,key) from aws secret manager 
+- Run above terraform command once again
+- Run below command in automation/terraform/monitoring folder: 
+
+```
+    ~/.local/bin/cortextool rules sync --rule-files alert_rules.yml --id ... --address ... --key ... 
+```
+
+## Alert Status
+
+#### GrafanaCloud Config
+
+To view the current testnet alerting rules config or verify that changes were applied correctly following a deployment, visit this Grafanacloud rules config [site](https://o1testnet.grafana.net/alerting/list?search=datasource:grafanacloud-o1testnet-prom).
+
+**Note:** ensure the datasource is set to `grafanacloud-o1testnet-prom` to access the appropriate ruleset. 
+
+#### Alertmanager UI
+
+Alerting rule violations can also be viewed in Grafanacloud's Alertmanager [UI](https://alertmanager-us-central1.grafana.net/alertmanager/#/alerts). This site provides an overview of all violating rule conditions in addition to rules that have been silenced.
+
+#### PagerDuty
+
+PagerDuty is O(1) Lab's primary alert receiver and is currently configured with a single [service](https://o1labs.pagerduty.com/service-directory/PY2JUNP) for monitoring Mina testnet deployments. This service receives alert notifications requiring attention from the development team to assist in repairing issues and restoring network health.
+
+For more information, reach out to [#reliability-engineering](https://discord.com/channels/484437221055922177/610580493859160072) on Mina's Discord channel with questions etc.
+
+## HowTo
+
+#### Silence Alerts
+
+* Alertmanager alert silencing: create new alert silences using either the [Alertmanager](https://alertmanager-us-central1.grafana.net/alertmanager/#/silences/new) or [Grafanacloud](https://o1testnet.grafana.net/a/grafana-alerting-ui-app/?tab=silences&alertmanager=grafanacloud-o1testnet-alertmanager) UI.
+* Pagerduty alert suppression: see [guide](https://support.pagerduty.com/docs/event-management#suppressing-alerts)
+
+##### Creating new silences
+
+When creating new alert silences (from the above links or otherwise), you'll likely want to make use of the AlertManager's `Matchers` construct, which basically consists of a set of key-value pairs used to target the alert to silence. For example, if silencing the "LowFillRate" alert currently firing for testnet *devnet*, you would create a new silence with individual `Matchers` for the alert name and testnet like the following:
```

Comment:
```suggestion
When creating new alert silences (from the preceding links or otherwise), you'll likely want to make use of the AlertManager `Matchers` construct that basically consists of a set of key-value pairs used to target the alert to silence. For example, to silencine the "LowFillRate" alert currently firing for testnet *devnet*, 
 create a new silence with individual `Matchers` for the alert name and testnet like the following:
```